### ISSUE TITLE
add entry cost to Safari zone

### ISSUE THREAD

**Issue by Claygame** [pokeclicker-dev/pokeclicker#55]
some kind of entry fee for safari zone would be nice, maybe some Tokens, definitely not quest points, i talked to nika and he also agreed that some fee for entry would be good.

**Comment by RedSparr0w** [comment]
How much do you think the entry fee should be?

If you leave before spending all your balls, should you have to pay again (start with 30 balls again)?

**Comment by Popoteguitou** [comment]
1000 token seem nice to me, and in the original game, when you quit the safari zone, you lose your remaining balls...

**Comment by Claygame** [comment]
yeah 1000 tokens seems fair and if you leave before using all balls then you lose them. Seems only logical and easier to do.
